src/NUnitEngine/nunit.engine/Services/ExtensionAssembly.cs

```diff
@@ -50,7 +48,7 @@ public AssemblyDefinition Assembly
                     resolver.AddSearchDirectory(System.IO.Path.GetDirectoryName(System.Reflection.Assembly.GetExecutingAssembly().Location));
                     var parameters = new ReaderParameters() { AssemblyResolver = resolver };
 
```

Comment:
Here, `ReaderParameters` was constructed but not passed in to `ReadAssembly()`. This still compiled because of an overload. I presumed passing the object in was the intention.